formats/gltf/material_tracker.go

```diff
@@ -0,0 +1,23 @@
+package gltf
+
+// materialEntry tracks a unique material and its corresponding GLTF material index
+type materialEntry struct {
+	polyMaterial PolyformMaterial
+	gltfMaterial Material
+	index        int
+}
```

Comment:
I don't think `gltfMaterial` is used

```suggestion
type materialEntry struct {
	material PolyformMaterial
	index    int
}
```